src/category_theory/limits/shapes/finite_products.lean

```diff
@@ -43,11 +43,8 @@ instance has_finite_products_of_has_finite_limits [has_finite_limits C] :
 
 instance has_fintype_products [has_finite_products C] (ι : Type w) [finite ι] :
```

Comment:
As the assumption changes, we may change the name?
```suggestion
instance has_limits_of_shape_discrete_finite_of_has_finite_products
  [has_finite_products C] (ι : Type w) [finite ι] :
```
(Same for `has_fintype_coproducts`.)

Reply:
I merged this instance with another one.